internal/service/streamprivatelinkendpoint/data_source.go

```diff
@@ -0,0 +1,55 @@
+//nolint:gocritic
+package streamprivatelinkendpoint
```

Comment:
as this is a new resource, have you considered working in master? i can give more info if you need it